pages/news/admin.py

```diff
@@ -0,0 +1,40 @@
+from django.contrib import admin
+from .models import News
+
+
+@admin.register(News)
+class NewsAdmin(admin.ModelAdmin):
+    """Admin interface for News model.
+
+    Provides a comprehensive admin interface for managing news
+    with organized fieldsets, search functionality, and filtering.
+
+    Features:
+        - List display with title, slug, status, and creation date
+        - Filtering by active status and creation date
+        - Search by name and summary
```

Comment:
```suggestion
        - Search by title and summary
```
Just a small mismatch with line 22, ensure that both are `title` or `name`, depending on which one is correct